build-release-tools/lib/manifest.py

```diff
@@ -45,7 +45,16 @@ def __init__(self, file_path, git_credentials = None):
         self.parse_manifest()
 
     @staticmethod
-    def instance_of_sample(manifest_sample="manifest.json"):
+    def instance_of_sample(type="rackhd"):
+        if type == "rackhd":
+            manifest_sample = "manifest.json"
+        elif type == "pr-gate":
+            manifest_sample = "manifest-pr-gate.json"
+        elif type == "on-web-ui":
+            manifest_sample = "manifest-web-ui.json"
+        else:
+            raise KeyError("No manifest template for type: {0}".format(type))
+
         repo_dir = os.path.dirname(sys.path[0])
```

Comment:
According to this PR, if we need to add CI/CD pipeline for other repository, we have to modify the script to add a key-value, right?
I don't think it's a good idea...